mits/mits_suite_test.go

```diff
@@ -0,0 +1,61 @@
+/*
+   Copyright 2020 SUSE
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package mits_test
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/SUSE/minibroker-integration-tests/mits"
+)
+
+var testsConfig map[string]mits.TestConfig
```

Comment:
Bike shedding: I'd be in favour of a proper struct describing each test, instead of just a mapping.  That we get to type-check things.

```go
// in mariadb_test.go
type mariadbTestConfig struct {
	TestConfig
}
// in this file
type testConfig struct {
	MariaDBConfig mariadbTestConfig `yaml:"mariadb"`
	…
}
```

I'm not blocking a merge on this, though.

Reply:
I'll defer this to the next PR. Noted.

Reply:
A version more towards that: https://github.com/SUSE/minibroker-integration-tests/pull/3/commits/3ca9348f2d5dda0d37f0f10bf017ce5b6bcec722.

If we need specialized structs for each, we can do itlater.